### ISSUE TITLE
name "socketio: is not defined

### ISSUE THREAD

**Issue by v479038280** [zauberzeug/nicegui#1208]
### Description

I got this error when I turn "on_air=True",here is the complete error message .I'm using python 3.11.4  on windows10.This might be a bug?
```
`name 'socketio' is not defined
Traceback (most recent call last):
  File "D:\Program Files\Python3114\Lib\site-packages\nicegui\air.py", line 107, in connect
    await self.relay.connect(
  File "D:\Program Files\Python3114\Lib\site-packages\socketio\asyncio_client.py", line 163, in connect
    raise exceptions.ConnectionError(
socketio.exceptions.ConnectionError: One or more namespaces failed to connect

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\Program Files\Python3114\Lib\site-packages\nicegui\background_tasks.py", line 55, in _handle_task_result
    task.result()
  File "D:\Program Files\Python3114\Lib\site-packages\nicegui\air.py", line 112, in connect
    except socketio.exceptions.ConnectionError:
           ^^^^^^^^
NameError: name 'socketio' is not defined`
```

**Comment by falkoschindler** [comment]
Sorry, @v479038280. This is a bug we accidentally introduced in 1.3.4 and fixed in 1.3.5.

**Comment by v479038280** [comment]
> I assume this issue is resolved. @v479038280 Feel free to re-open it if this is not the case.

I upgrade nicegui to 1.3.5, and on_air=True, here is the result
![微信截图_20230721045530](https://github.com/zauberzeug/nicegui/assets/25096045/19be7c74-473a-434b-9e44-f466cfbb9e79)

**Comment by rodja** [comment]
@v479038280, remote url not showing up is a similar issue as https://github.com/zauberzeug/nicegui/issues/1279 which we hope to fix that with the next release 1.3.7.
